spec/requests/shows_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe ShowsController, type: :feature do
+  describe "GET shows#index" do
+    it 'renders index and has the header' do
+      visit '/'
+      expect(page).to have_text 'Smartflix'
```

Comment:
you could group all these specs into single spec, you run same page multiple times which takes time if you had thousands of feature specs. Best if you group expectations in the same it block

Reply:
Only create another IT block if you are testing some other behaviour on the same page under different conditions.